npm run validate errors out at npm run check-coverage

Am I the only one finding error for `npm run validate`, to be precise `npm run check-coverage`

`coverage` directory does exist, I have verified it with `npm run cover`.

```
ERROR: check-coverage: None-Zero Exit(1);

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "validate"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! eslint-find-new-rules@0.0.0-semantically-released validate: `npm-run-all --parallel lint cover --sequential check-coverage`
npm ERR! Exit status 1
```

---

```
吽 npm run check-coverage

> eslint-find-new-rules@0.0.0-semantically-released check-coverage /Users/sabandyo/GitHub/eslint-find-new-rules
> nyc check-coverage --statements 71 --branches 33 --functions 100 --lines 71

ERROR: No coverage files found.

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "check-coverage"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! eslint-find-new-rules@0.0.0-semantically-released check-coverage: `nyc check-coverage --statements 71 --branches 33 --functions 100 --lines 71`
npm ERR! Exit status 1
```

Thanks!!

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,7 +14,6 @@ before_script:
   - npm prune
 script:
   - npm run validate
-  - npm run check-coverage
 after_success:
   - 'curl -Lo travis_after_all.py https://git.io/travis_after_all'
   - python travis_after_all.py

--- a/package.json
+++ b/package.json
@@ -4,13 +4,13 @@
   "description": "Use this to save yourself some time when working on a webpack configuration.",
   "main": "index.js",
   "scripts": {
-    "cover": "nyc --reporter=lcov --reporter=text npm t",
+    "cover": "nyc --reporter=lcov --reporter=text ava",
     "test": "ava",
     "lint": "eslint .",
     "check-kentcdodds": "./bin.js eslint-config-kentcdodds",
     "update-contributors": "all-contributors generate",
     "commit": "git-cz",
-    "validate": "npm-run-all --parallel lint cover",
+    "validate": "npm-run-all --parallel lint cover --sequential check-coverage",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "check-coverage": "nyc check-coverage --statements 71 --branches 33 --functions 100 --lines 71",
     "report-coverage": "cat ./coverage/lcov.info | node_modules/.bin/codecov"

